scripts/inference

```diff
@@ -17,9 +17,11 @@ then
     myDir="."
 fi
 
+ROOT=$(cd ${myDir}/../../ && pwd)
```

Comment:
Maybe this is JSR308? I thought ROOT is the CFI project root.

Reply:
Sure, will change the name.